src/controllers/connectionManager.ts

```diff
@@ -1521,6 +1528,23 @@ export default class ConnectionManager {
         return await connectionCompletePromise;
     }
 
+    public onActiveConnectionsChanged(listener: () => void): void {
```

Comment:
NIT: jsdoc comments for public methods.

Reply:
Added the following JS doc comment
> /**
     * Registers a listener that is triggered when the active connections change.
     *
     * @param listener - A callback function to be invoked when the "activeConnectionsChanged" event occurs.
     */